### PR TITLE
Mois 44 GitHub actions - Revisited

### PR DESCRIPTION
**Pull request by zlatax**
- Cypress tests now run on emulated firebase server.
- Test artifacts (screenshots for errors, videos per test case) are uploaded and stored as Workflow artifacts per push/PR - This can be changed to push/PR only for main branch if deemed necessary. The artifacts per push/PR can help with providing evidence of testing for reviewers and developer making the change.

### CODE REVIEW

**Review by edsonli1999 — APPROVED** [review]
LGTM!

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -4,11 +4,14 @@ name: CI
 
 # Controls when the workflow will run
 on:
-  # Triggers the workflow on push or pull request events but only for the "main" branch
+  # # Triggers the workflow on push or pull request events but only for the "main" branch
```
Comment (IkePureza) [inline .github/workflows/main.yml:7]: You have to revert this right before merging to main?

**Review by IkePureza — APPROVED** [review]
Ay good stuff man, this looks good!

File: .github/workflows/main.yml (hunk 1 above)
Comment (zlatax) [reply .github/workflows/main.yml:7]: Will do!

**Pull request merged**